CHANGELOG.md

```diff
@@ -17,6 +16,7 @@
 * Show new request action for checked out items. Fixes UIIN-1188.
 * Show new request action for on order items. Fixes UIIN-1187.
 * Add `Edit in quickMARC` link disabled state condition. Refs UIIN-1191
+* Possible error for Source=FOLIO instances has been fixed. Refs UIIN-1190.
```

Comment:
Could you please place it at the bottom right after ` Increment `@folio/stripes` to `v5`, plus corresponding dev-dep increments.`?

Reply:
done